.github/renovate.json5

```diff
@@ -55,33 +54,33 @@
         'redis', // Requires manual upgrade and sync with Sidekiq version
```

Comment:
Same here, `sidekiq` depends on `redis < 5`. I wonder if Renovate will offer the update as we have a dependency preventing upgrading to it.